buildAndReleaseTask/task.json

```diff
@@ -76,6 +76,15 @@
             "required": false,
             "helpMarkDown": "If you want Cobertura reports to be generated for the test execution, enable this.",
             "groupName": "codeCoverage"
+        },
+		{
+            "name": "enableAzure",
+            "type": "boolean",
+            "label": "Enable Azure SQL",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "If you want to get the result from a Azure SQL Server, enable this.",
```

Comment:
I was also thinking about requesting this info from the user as a Dropdown list. What do you think?

This way, we could ask for something like:

"SQL Server technology/service type":
* SQL Server On Premises
* Azure SQL

And then we'd handle this on the scripts.